README.md

```diff
@@ -0,0 +1,106 @@
+</ol>
+
+![](https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-12-35/MediaObjects/12859_2010_Article_5075_Fig1_HTML.jpg?as=webp)
+
+Selected Venn diagram special cases and Euler diagrams drawn by the VennDiagram package.
+
+- **Row 1, column 1**: automatically drawn, customizable lines that optimize display of partial areas when individual partial areas become too small in two-set Venn diagrams.
+- **Row 1, column 2**: a two-set Euler diagram showing total inclusion of one of the sets.
+- **Row 1, column 3**: a two-set Euler diagram showing two distinct sets.
+- **Row 2, column 1**: a three-set Euler diagram where one set has no discrete elements.
+- **Row 2, column 2**: a three-set Euler diagram where one set has no discrete elements is totally included in one of the other two sets.
+- **Row 2, column 3**: a three-set Euler diagram where two sets have no discrete elements and are included in a larger third set.
+- **Row 3, column 1**: a three-set Euler diagram showing total inclusion of two sets that are distinct from the third set.
+- **Row 3, column 2**: a three-set Euler diagram where one set is totally included in another set, which is itself totally included in the third set.
+- **Row 3, column 2**: a three-set Euler diagram showing three distinct sets.
+
+![](https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-12-35/MediaObjects/12859_2010_Article_5075_Fig2_HTML.jpg?as=webp)
+
+A side-by-side comparison of an Euler diagram and a Venn diagram for the same hypothetical sets.
+
+<!-- to get A, B -->
+<ol type="a">
+    <li>The Euler diagram shows only non-zero areas and can therefore be more graphically accurate.</li>
+    <li>The Venn diagram shows the non-existent area as an area with zero content. Though this is not graphically accurate, it preserves the recognisability of a Venn diagram.</li>
+</ol>
+
+![](https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-12-35/MediaObjects/12859_2010_Article_5075_Fig3_HTML.jpg?as=webp)
+
+## Resources
+
+Available resources for VennDiagram include the package [CRAN page](https://cran.r-project.org/web/packages/VennDiagram/index.html) and [reference manual](https://cran.r-project.org/web/packages/VennDiagram/VennDiagram.pdf).
+
+## Getting help
+
+For guidance or support with VennDiagram check out [Discussions](https://github.com/uclahs-cds/public-R-VennDiagram/discussions)
+
+See [Issues](https://github.com/uclahs-cds/public-R-VennDiagram/issues) to submit bugs, suggest new features or view current works
+
+[Pull requests](https://github.com/uclahs-cds/public-R-VennDiagram/pulls) are also open for discussion.
+
+## Citation information
+
+Chen, H., Boutros, P.C. VennDiagram: a package for the generation of highly-customizable Venn and Euler diagrams in R. BMC Bioinformatics 12, 35 (2011). https://doi.org/10.1186/1471-2105-12-35
```

Comment:
```suggestion
To cite package `VennDiagram` in publications use:

  Chen, H., Boutros, P.C. VennDiagram: a package for the generation of highly-customizable Venn and
  Euler diagrams in R. BMC Bioinformatics 12, 35 (2011). https://doi.org/10.1186/1471-2105-12-35

A BibTeX entry for LaTeX users is

```
@Article{,
    title = {VennDiagram: a package for the generation of highly-customizable Venn and Euler diagrams in R},
    journal = {BMC Bioinformatics},
    doi = {10.1186/1471-2105-12-35},
    url = {https://doi.org/10.1186/1471-2105-12-35},
    volume = {12},
    number = {35},
    year = {2011},
    month = {January},
    day = {26},
    issn = {1471-2105},
    author = {Hanbo Chen and Paul C Boutros},
  }
```
```